tests/test_black.py

```diff
@@ -1764,7 +1776,7 @@ def test_gitignore_used_as_default(self) -> None:
         src = [base / "b/"]
         assert_collected_sources(src, expected, extend_exclude=r"/exclude/")
 
-    @patch("black.find_project_root", lambda *args: THIS_DIR.resolve())
+    @patch("black.find_project_root", lambda *args: (THIS_DIR.resolve(), "None"))
```

Comment:
```suggestion
    @patch("black.find_project_root", lambda *args: (THIS_DIR.resolve(), None))
```
This repeats a bunch of times.